sycl/include/CL/sycl/backend/level_zero.hpp

```diff
@@ -9,8 +9,7 @@
 #pragma once
 
 #include <CL/sycl.hpp>
-// This header should be included by users.
-//#include <level_zero/ze_api.h>
+#include <level_zero/ze_api.h>
```

Comment:
Yes, I remember that we changed it back to avoid compilation fail on app/systems where sycl/backend/level_zero.hpp is included, but where there is no real use of Level-Zero interop. But there was such problem just because we didn't install properly Level Zero headers with dpcpp compiler. As a result of this commit Level Zero headers are installed to the standard place (include/sycl) where DPCPP compiler looks for headers (sycl, opencl, level zero).

Motivation for this change:
1. To be similar with Opencl. Currently we install opencl headers to the include/sycl.
2. To enable lit tests for level zero interoperability. Without this commit it is required to include headers in this specific order:
#include <level_zero/ze_api.h>
#include <CL/sycl/backend/level_zero.hpp>
and this order is not aligned with clang-format requirements.
3.  From the user's point of view it is strange that  headers must be included in some specific order.

Reply:
Reverted this change in this PR.